lua/tree-sitter-jinja2.lua

```diff
@@ -1,16 +1,37 @@
 local M = {}
 
+function M.register_parser()
+  local ok, parsers = pcall(require, "nvim-treesitter.parsers")
+  if not ok then
+    return
+  end
```

Comment:
I’d rather not silently drop errors. This makes things hard to debug usually.

```suggestion
  local parsers = require("nvim-treesitter.parsers")
```